src/main/java/io/github/bbortt/event/planner/repository/RoleRepository.java

```diff
@@ -1,16 +1,32 @@
 package io.github.bbortt.event.planner.repository;
 
+import io.github.bbortt.event.planner.domain.Project;
 import io.github.bbortt.event.planner.domain.Role;
+import java.util.List;
+import javax.validation.constraints.NotNull;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 /**
  * Spring Data  repository for the Role entity.
  */
-@SuppressWarnings("unused")
 @Repository
 public interface RoleRepository extends JpaRepository<Role, String> {
-    @Query("FROM Role r where r.name = io.github.bbortt.event.planner.security.RolesConstants.ADMIN")
+    @Query("from Role r where r.name = io.github.bbortt.event.planner.security.RolesConstants.ADMIN")
     public Role roleAdmin();
```

Comment:
this can probably be removed/replaced by `RoleRepository#findById(RolesConstants)`. `name` is the primary key of thge `Role` entity.